policy/modules/system/udev.te

```diff
@@ -170,6 +170,7 @@ init_start_all_units(udev_t)
 init_stop_all_units(udev_t)
 # for hdparm init script run by udev
 init_get_script_status(udev_t)
+init_list_runtime(udev_t)
```

Comment:
This should probably be part of the existing auth_use_nsswitch() call below. There is a PR that addresses that:
https://github.com/SELinuxProject/refpolicy/pull/269